entries/w/wordpress.com.json

```
@@ -3,12 +3,13 @@
     "domain": "wordpress.com",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "custom-software"
```

I see you tried to add security keys. Here's how to do that:
```suggestion
      "u2f"
```